prereise/gather/demanddata/transportation_electrification/smart_charging.py

```diff
@@ -166,6 +287,15 @@ def smart_charging(
                 # copy one vehicle information to the block
                 individual = newdata.iloc[i : i + total_trips].copy()
 
+                individual_vmt += float(
+                    individual.iloc[
+                        0, individual.columns.get_loc("total vehicle miles traveled")
+                    ]
+                )
+                individual_trip_miles += float(
+                    individual.iloc[:, individual.columns.get_loc("trip_miles")].sum()
```

Comment:
Maybe it is equivalent to do
```python
individual["trip_miles"].sum()
```

Reply:
Same: This was something I added I believe, so if this is equivalent, I'm open to the change.